fe/fe-core/src/main/java/com/starrocks/planner/JoinNode.java

```diff
@@ -195,11 +208,12 @@ public void buildRuntimeFilters(IdGenerator<RuntimeFilterId> runtimeFilterIdIdGe
                 // push down rf to left child node, and build it only when it
                 // can be accepted by left child node.
                 rf.setBuildExpr(left);
-                if (getChild(0).pushDownRuntimeFilters(rf, right)) {
+                if (getChild(0).pushDownRuntimeFilters(rf, right, probePartitionByExprs)) {
```

Comment:
if the grf is a one-column grf generated from one-equal HJ or a multi-column local rf, is probePartitionByExprs empty?